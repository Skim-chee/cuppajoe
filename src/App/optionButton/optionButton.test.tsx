import React from 'react';
import { shallow } from 'enzyme';
import OptionButton from './optionButton';
import toJson from 'enzyme-to-json';

describe('Header', () => {
	const Props = {
		feeling: 'chill',
		option: 1,
		activeOption: 1
	};
	const wrapper = shallow(<OptionButton {...Props} />);

	it('render the component', () => {
		expect(wrapper.length).toEqual(1);
	});

	it('matches the snapshot', () => {
		expect(toJson(wrapper)).toMatchSnapshot();
	});
});
