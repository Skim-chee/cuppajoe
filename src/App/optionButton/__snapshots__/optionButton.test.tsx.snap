// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header matches the snapshot 1`] = `
<div>
  <input
    className="f1wt0yvj"
    name="1"
    type="radio"
    value="chill"
  />
  <label
    className="f1uscnny fa7andv"
  />
  1
   
  1
</div>
`;
